include/modem/at_cmd_parser.h

```diff
@@ -127,7 +127,7 @@ enum at_cmd_type {
  *
  * @return Command type.
  */
-enum at_cmd_type at_parser_cmd_type_get(const char *at_cmd);
+enum at_cmd_type at_parser_at_cmd_type_get(const char *at_cmd);
```

Comment:
No, because this commit has to be bisectable and correct by itself. If the change was in a separate commit the documentation build would fail in the AT parser commit.